src/dbSta/src/MakeDbSta.cc

```diff
@@ -29,11 +29,6 @@ namespace ord {
 
 using sta::dbSta;
 
-dbSta* makeDbSta()
```

Comment:
Why is this moved?

Reply:
In order to construct this from tests the linker needs to see this function in the lib version of dbsta instead of the binary version. 

The new operator itself isn't callable directly because of the abstract gui class pointers.